### DISCUSSION TITLE
Search error by client error UUID

### DISCUSSION THREAD

**Discussion by Jellyfrog** [spatie/flareapp.io-roadmap#59] [Ideas]
We can get the assigned UUID with `\Facade\Ignition\Facades\Flare::sentReports()->latestUuid()` but you can't search for it,
Making it really hard to help a customer saying "I got this error code"

**Comment by AlexVanderbist** [comment]
We've recently added this. You can now filter errors and occurrences using the `tracking_uuid` filter: https://flareapp.io/docs/flare/general/searching-errors
